gae/js/app/views/staffposition.js

```diff
@@ -0,0 +1,126 @@
+define(
+    [
+        'backbone', 'backform',
+        'app/views/staffposition_rates',
+        'text!app/templates/staffposition.html',
+    ],
+    function(Backbone, Backform, RatesView, template) {
+
+        var StaffPositionFlow = Backbone.View.extend({
+            el: '#simple-form-view',
+            events: {
+                'click button[name=saveStaffPosition]': 'saveStaffPosition',
+            },
+            initialize: function(options) {
+                var self = this;
+                this.options = options;
+                self.staffposition = options.staffposition;
+                self.template = _.template(options.template);
+                self.loading = options.loading;
+                self.hourly_rates = self.staffposition.get('hourly_rates');
```

Comment:
Probably don't need to save hourly_rates on the view here.  Can just access them via the model.